Networking/Networking/Remote/ShippingLabelRemote.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+/// Shipping Labels Remote Endpoints.
+public final class ShippingLabelRemote: Remote {
+    /// Generates shipping label data for printing.
+    /// - Parameters:
+    ///   - siteID: Remote ID of the site that owns the shipping label.
+    ///   - shippingLabelID: Remote ID of the shipping label.
+    ///   - paperSize: Paper size string (current options are "label", "legal", and "letter").
+    ///   - completion: Closure to be executed upon completion.
+    public func printShippingLabel(siteID: Int64,
+                                   shippingLabelID: Int64,
+                                   paperSize: String,
+                                   completion: @escaping (Result<ShippingLabelPrintData, Error>) -> Void) {
+        let parameters = [
+            ParameterKey.paperSize: paperSize,
+            ParameterKey.labelIDCSV: String(shippingLabelID),
+            ParameterKey.cationCSV: "",
```

Comment:
nit
```suggestion
            ParameterKey.captionCSV: "",
```

Reply:
hah nice catch! fixed in https://github.com/woocommerce/woocommerce-ios/pull/3155/commits/562395447db31ea72d79aa601f15a7570421eec1